luigi/scheduler.py

```diff
@@ -840,6 +842,10 @@ def add_task(self, task_id=None, status=PENDING, runnable=True,
         if task is None or (task.status != RUNNING and not worker.enabled):
             return
 
+        # if the task was completed only recently, do not run the same task again
```

Comment:
Describe what you actually do, the implementation, not why you do it. :)

     # Ignore claims that the task is PENDING if it very recently was marked as DONE.

Maybe?

Reply:
Thanks, changed to the proposed version